examples/crd/crd-v1alpha2.yaml

```diff
@@ -9,3 +9,26 @@ spec:
     kind: TFJob
     singular: tfjob
     plural: tfjobs
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            tfReplicaSpecs:
+              properties:
+                Worker:
+                  properties:
+                    replicas:
+                      type: integer
```

Comment:
Why is the minimum 1? Shouldn't the minimum be 0 for Worker & PS since they aren't required?
Even chief should have minimum 0 because in some cases we will have N workers and the first worker will be the chief and there will be no explict chief.

Reply:
If there is no worker and PS, the check will not happen. Thus this is for the case that we have the worker/ps/chief definition.

Reply:
Can we add a comment to that effect?

Reply:
Thanks. Can we put a comment to that effect?